src/modules/populateComment.js

```diff
@@ -0,0 +1,177 @@
+
+  const popupCommentSection = document.createElement('article');
+  const h35 = document.createElement('h3');
+  h35.setAttribute('class', 'detail detail3');
+  h35.innerText = 'Comments';
+  const span1 = document.createElement('span');
+  span1.innerText = '(2)';
+  h35.appendChild(span1);
+  const dateComment1 = document.createElement('div');
+  dateComment1.setAttribute('class', 'date_comment');
+  const b1 = document.createElement('b');
+  b1.setAttribute('class', 'detail_p');
+  b1.innerText = '03/11/2021';
+  const p5 = document.createElement('p');
+  p5.setAttribute('class', 'detail_p');
+  p5.innerText = "Alex: I'd love to buy it";
+  dateComment1.appendChild(b1);
+  dateComment1.appendChild(p5);
+  const dateComment2 = document.createElement('div');
+  dateComment2.setAttribute('class', 'date_comment');
+  const b2 = document.createElement('b');
+  b2.setAttribute('class', 'detail_p');
+  b2.innerText = '03/12/2021';
+  const p6 = document.createElement('p');
+  p6.setAttribute('class', 'detail_p');
+  p6.innerText = 'I love';
+  dateComment2.appendChild(b2);
+  dateComment2.appendChild(p6);
+  popupCommentSection.appendChild(h35);
+  popupCommentSection.appendChild(dateComment1);
+  popupCommentSection.appendChild(dateComment2);
+
+  const popupForm = document.createElement('form');
+  popupForm.setAttribute('class', 'popup_form');
+  popupForm.setAttribute('id', 'form');
+  const h36 = document.createElement('h3');
+  h36.setAttribute('class', 'detail detail4');
+  h36.innerText = 'Add a Comment';
+  const username = document.createElement('input');
+  username.setAttribute('name', 'username');
+  username.setAttribute('class', 'detail_p');
+  username.setAttribute('value', '');
+  username.setAttribute('id', 'username');
+  username.setAttribute('minlength', '5');
+  username.setAttribute('placeholder', 'Your name');
+  username.setAttribute('required', true);
+  username.setAttribute('type', 'text');
+  const insight = document.createElement('input');
+  insight.setAttribute('name', 'insight');
+  insight.setAttribute('class', 'detail_p');
+  insight.setAttribute('value', '');
+  insight.setAttribute('id', 'insight');
+  insight.setAttribute('minlength', '1');
+  insight.setAttribute('placeholder', 'Your name');
+  insight.setAttribute('required', true);
+  insight.setAttribute('type', 'text');
+  const formbutton = document.createElement('button');
+  formbutton.setAttribute('type', 'submit');
+  formbutton.setAttribute('class', 'button1');
+  formbutton.setAttribute('id', 'comment');
+  formbutton.innerText = 'Comment';
+
+  popupForm.appendChild(h36);
+  popupForm.appendChild(username);
+  popupForm.appendChild(insight);
+  popupForm.appendChild(formbutton);
+
+  popup.appendChild(upperpopup);
+  popup.appendChild(middlepopup);
+  popup.appendChild(popupCommentSection);
+  popup.appendChild(popupForm);
+
+  thebody.appendChild(popup);
+
+  closeicon.addEventListener('click', () => {
```

Comment:
- When the close button is pressed the page scroll to the top

https://user-images.githubusercontent.com/68030297/197026462-b108178e-087f-4bf8-9eb9-2979f0b48074.mp4

- [ ] Since the close tag is an anchor "link" this will cause the page to scroll to the top, so to prevent that from happening we can do something like the example below:

```javascript
closeicon.addEventListener('click', (e) => { 
   e.preventDefault();
   ...
}
```